pkg/ttnpb/field_mask_validation.go

```diff
@@ -55,6 +56,7 @@ var isEndDeviceWriteFieldPaths = []string{
 	"version_ids.firmware_version",
 	"version_ids.hardware_version",
 	"version_ids.model_id",
+	"picture",
```

Comment:
IDEM

Reply:
Yeah this is really fine, we only allow updating the entire picture object.